.changeset/lemon-years-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": major
```

Comment:
Don't think this change would need a major version update. Should be minor or patch depending on the Wrangler team's recommendation.

Reply:
Sorry, this was a mistake on my part